frontend/gpay_agent/ios/Runner/Info.plist

```diff
@@ -41,5 +41,13 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>NSLocationWhenInUseUsageDescription</key>
+    <string>This app needs access to location when open.</string>
+
+    <key>NSLocationAlwaysUsageDescription</key>
+    <string>This app needs access to location when in the background.</string>
+
+    <key>NSLocationAlwaysAndWhenInUseUsageDescription</key>
```

Comment:
Based on above comment you may only need one of these

Reply:
Removed not required permissions